Add proposed sensors to WeatherFlow doc (#45168)
home-assistant/home-assistant.github.io: source/_integrations/weatherflow.markdown

## Sensors

> This integration will expose the following sensors:

Reviewer: ```suggestion
This {% term integration %} will expose the following sensors:
```

- **Wind speed**
  - **Description**: Wind speed at the station.

### Precipitation sensors

- **Precipitation**
  - **Description**: Precipitation (accumulated over the previous minute).

- **Precipitation intensity**
  - **Description**: Precipitation intensity ([extrapolated](https://weatherflow.github.io/Tempest/api/derived-metric-formulas.html#rain-rate) from the accumulation over the previous minute).

- **Precipitation type**
  - **Description**: Precipitation type (`none`, `rain`, `hail`, `rain_hail`).

### Lightning sensors

- **Lightning average distance**
  - **Description**: Average distance of recently detected lightning strikes.

- **Lightning count**
  - **Description**: Count of lightning strikes in the past minute.

- **Lightning last distance**
  - **Description**: Distance to the most recent detected lightning strike.

- **Lightning last energy**
  - **Description**: Energy estimate for the most recent detected lightning strike, as reported by the station.

- **Lightning last strike**
  - **Description**: Timestamp of the most recent detected lightning strike; unlike the `Lightning strike` event entity below, this sensor stores the last recorded strike time.

> ### Solar & Light sensors

Reviewer: ```suggestion
### Solar and light sensors
```

- **Illuminance**
  - **Description**: Amount of visible light received, measured in lux.

- **Irradiance**
  - **Description**: Total solar radiation received, measured in W/m².

- **UV index**
  - **Description**: Intensity of ultraviolet radiation received.

### Diagnostic sensors

Additionally the following diagnostic sensors are available:

- **Battery (percentage)**
  - **Description**: Station battery (percentage).

- **Battery voltage**
  - **Description**: Battery voltage of station.

- **Signal strength**
  - **Description**: Signal strength between station and hub.

- **Uptime**
  - **Description**: Uptime of station.

## Event entities

> The WeatherFlow Tempest station also sends out event triggers when it starts raining and when there is a lightning strike near by.

Reviewer: ```suggestion
The WeatherFlow Tempest station also sends event triggers when it starts raining and when there is a lightning strike nearby.
```

- **Wind speed**
  - **Description**: Wind speed at the station.

### Precipitation sensors

- **Precipitation**
  - **Description**: Precipitation (accumulated over the previous minute).

- **Precipitation intensity**
  - **Description**: Precipitation intensity ([extrapolated](https://weatherflow.github.io/Tempest/api/derived-metric-formulas.html#rain-rate) from the accumulation over the previous minute).

- **Precipitation type**
  - **Description**: Precipitation type (`none`, `rain`, `hail`, `rain_hail`).

### Lightning sensors

- **Lightning average distance**
  - **Description**: Average distance of recently detected lightning strikes.

- **Lightning count**
  - **Description**: Count of lightning strikes in the past minute.

- **Lightning last distance**
  - **Description**: Distance to the most recent detected lightning strike.

- **Lightning last energy**
  - **Description**: Energy estimate for the most recent detected lightning strike, as reported by the station.

- **Lightning last strike**
  - **Description**: Timestamp of the most recent detected lightning strike; unlike the `Lightning strike` event entity below, this sensor stores the last recorded strike time.

> ### Solar and Light sensors

Reviewer: ```suggestion
### Solar and light sensors
```
